README.rst

```diff
@@ -63,14 +63,69 @@ Publishing
     git tag 0.5.1
     git push origin 0.5.1
 
-4. Register the package::
+``.pypirc`` file
+-----------------
+
+To ensure publication and testing of publications are submitted to the correct
+servers (with credentials) create a ``.pypirc`` file in your ``HOME`` directory.
+This file should looks as follows:
+
+::
+
+    [distutils]
+    index-servers=
+        pypi
+        testpypi
+
+    [testpypi]
+    repository = https://testpypi.python.org/pypi
+    username = <username>
+    password = <password>
+
+    [pypi]
+    repository = https://pypi.python.org/pypi
+    username = <username>
+    password = <password>
+
+
+Note that ``testpypi`` and ``pypi`` require separate registration.
+
+Testing Publication
+-----------------
+
+1. Ensure you have tagged the master repository with the according to the
+tagging instructions above.
+
+2. Register the package::
+
+    python setup.py register -r testpypi
+
+3. Package source and wheel distributions::
+
+    python setup.py sdist bdist_wheel
+
+4. Upload to PyPI with twine::
+
+    twine upload dist/* -r testpypi
+
+5. Test that you can install the package from testpypi::
+
+    pip install -i https://testpypi.python.org/pypi analyzere
+
+Publishing
+-----------
+
+1. Ensure you have tagged the master repository with the according to the
```

Comment:
`Ensure you have tagged the master repository with the according to the`

=> 

`Ensure you have tagged the master repository according to the`